### ISSUE TITLE
Provide an example (specs?) of DataStore using Reddis for caching

### ISSUE THREAD

**Issue by laser** [stormpath/stormpath-sdk-ruby#34]
Blocked on https://github.com/stormpath/stormpath-sdk-ruby/issues/33

Once the in-memory cache implementation has been completed, provide an example of how the developer can use Reddis instead.

**Comment by tilo** [comment]
@omgitstom there are no specs for how to use and configure the Redis server for the Stormpath Ruby SDK -- e.g. how / where should the Redis URL and driver be configured?

**Comment by cilim** [comment]
@tilo currently there's a bug with setting the custom redis configuration. The store options hash isn't properly passed to the `RedisStore` class that instantiates the Redis server.

I've found a solution, but it will be available tomorrow or the day after that (with the specs and all)

**Comment by cilim** [comment]
@tilo, the bug has been fixed and new version of the Ruby SDK released. Please update it to `1.2.1` and notice the new ``store_opts`` hash that you can pass when instantiating the client and its cache store:

```ruby
client = Stormpath::Client.new(
  api_key_file_location: api_key_file_location,
  cache: {
    store: Stormpath::Cache::RedisStore,
    store_opts: {
      host: '10.0.1.1',
      port: 6380,
      db: 15
    }
  }
)
```

Please try it out, and if any new errors occur, feel free and open a new issue.
Thanks.

**Comment by cilim** [comment]
It can also be viewed in detail on the [Ruby SDK Documentation, configuration chapter](https://docs.stormpath.com/ruby/product-guide/latest/configuration.html#redis)
